OpenRA.Mods.Common/UpdateRules/Rules/SwitchShapePointsX.cs

```diff
@@ -0,0 +1,71 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class SwitchShapePointsX : UpdateRule
+	{
+		public override string Name { get { return "Switch positive and negative X values on HitShape points"; } }
```

Comment:
Are you 100% sure that `Y` isn't inverted too?

Reply:
Not in terms of end result in any case, on buildings (which are always facing 0/north), negative means "left", which matches how all our other offsets work too, afaik.